views/groups.ejs

```diff
@@ -48,8 +48,9 @@
         <div class="col-lg-3">
           <a href="/group/<%- encodeURI(groups[i]._id) %>">
             <div class="features-icons-item mx-auto mb-5 mb-lg-0 mb-lg-3">
-              <div class="features-icons-icon d-flex">
-                <img src="<%= groups[i].picture %>" class="img-thumbnail"></i>
+
+              <div class="d-flex">
+                <img src="/<%= groups[i].picture %>" class="img-thumbnail"></i>
```

Comment:
können wir einheitlich entscheiden wo wir denn jetzt der "/" hinbauen? in den code oder in die Daten.
@phillies willst du das noch einheitlich machen, bevor wir es mergen und live stellen?